packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -33,25 +34,52 @@ const Banners = () => {
         return transport?.outdated;
     };
 
+    const getMessageSystemBanner = (): Message | null => {
```

Comment:
Seems to have quite some shuffling around happening here, maybe memoizing that one too might be a good shout.